graphql-dgs-example-shared/src/test/java/com/netflix/graphql/dgs/example/shared/HelloDataFetcherTest.java

```diff
@@ -56,7 +56,7 @@ void getQueryWithInspectError() {
             queryExecutor.executeAndExtractJsonPath("{greeting}", "data.greeting");
             fail("Exception should have been thrown");
         } catch (QueryException ex) {
-            assertThat(ex.getMessage()).contains("Validation error of type FieldUndefined: Field 'greeting' in type 'Query' is undefined @ 'greeting'");
+            assertThat(ex.getMessage()).contains("Validation error (FieldUndefined@[greeting]) : Field 'greeting' in type 'Query' is undefined");
```

Comment:
See: https://github.com/graphql-java/graphql-java/pull/2878

Reply:
latest master already uses GraphQL Java 19.x

Reply:
Wonderful, just merged main back into this branch to take advantage of these updates.